infra/resources.bicep

```diff
@@ -408,6 +417,12 @@ module grafana './modules/container-app.bicep' = {
         storageType: 'NfsAzureFile'
         mountOptions: 'dir_mode=0777,file_mode=0777,uid=1000,gid=1000,mfsymlinks,nobrl,cache=none'
       }
+      {
```

Comment:
why does the Grafana dashboard need an Elastic Search volume mount?

Reply:
so that's just for troubleshooting... I had issues with elastic being blocked from accessing the share.
I'm thinking I could just use 1 share different paths, instead of 4 shares

Reply:
I kept them on separate file shares so that they would be completely isolated (just like the compute is isolated into different containers)

but I could also see just combining them together since there isn't likely to be security or performance concerns

what do you think?

Reply:
I think that would be simpler. If we use the sub directories, that's going to handle the isolation

Reply:
ok, but now you have to remove the extra file shares from the IaC and update the directory subpaths :)